### ISSUE TITLE
`wget` missing in the requirement?

### ISSUE THREAD

**Issue by sfc-gh-ywei** [evalplus/repoqa#44]
```
Traceback (most recent call last):                                                            
  File ".../.local/bin/repoqa.search_needle_function", line 5, in <module>
    from repoqa.search_needle_function import main                      
  File ".../.local/lib/python3.10/site-packages/repoqa/search_needle_function.py", line 11, in <module>
    from repoqa.compute_score import compute_score, save_json
  File ".../.local/lib/python3.10/site-packages/repoqa/compute_score.py", line 22, in <module>
    from repoqa.data import get_repoqa_data 
  File ".../.local/lib/python3.10/site-packages/repoqa/data.py", line 10, in <module>
    import wget                                                                                                                                                                              
ModuleNotFoundError: No module named 'wget'                                                                                                                                                  
```

**Comment by ganler** [comment]
You can temporarily solve it by pip install wget, Mr ywei. :)

**Comment by ganler** [comment]
Fixed by 88480ee01180ef97a83df891e022a07df0abebd6

Also updated pypi wheels
